cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -38,31 +43,53 @@ func NewCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 
 			var err error
-			o := scorecard.Options{}
+			o := scorecard.Options{
+				ServiceAccount: serviceAccount,
+				Namespace:      namespace,
+				List:           list,
+				BundlePath:     bundle,
+				OutputFormat:   output,
+				Cleanup:        cleanup,
+				WaitTime:       waitTime,
+			}
+			o.Client, err = scorecard.GetKubeClient(kubeconfig)
```

Comment:
ok, I found that the controller-runtime has a GetConfig(), I've updated the code to use that since it was used in the 'operator-sdk new' command.  I found that to support a --kubeconfig cobra flag with this library that I had to set the KUBECONFIG env var if --kubeconfig is specified.  I found an example of that in cmd/operator-sdk/run/local.go.